src/meta/api/src/schema_api_impl.rs

```diff
@@ -2379,6 +2381,12 @@ impl<KV: kvapi::KVApi<Error = MetaError>> SchemaApi for KV {
                 txn_req.if_then.extend(match_operations)
             }
 
+            if let Some(deduplicated_label) = req.deduplicated_label.clone() {
+                txn_req
+                    .if_then
+                    .push(build_upsert_table_deduplicated_label(deduplicated_label))
+            }
+
```

Comment:
Who will consume this label? 

I assume every SchemaApi method that updates the copied file must assert its label does not exists to proceed.